static/frontend/Caddyfile

```diff
@@ -0,0 +1,30 @@
+{
+	admin off
+	http_port 8080
+	https_port 8443
+
+	# Do not add the webserver's local certificate authority to the container's trust store
+	skip_install_trust
+}
+
+localhost {
+	# Issue self-signed certificates on-demand from local certificate authority
+	tls {
+		issuer internal
```

Comment:
Todo: this certificate `issuer` configuration will require updates if/when this configuration is used in combination with public ACME CA servers.